src/appsettings/settings.py

```diff
@@ -967,3 +974,41 @@ def check(self):
     def transform(self, value):
         """Transform each item in the list."""
         return tuple(self.inner_setting.transform(item) for item in value)
+
+
+class FileSetting(Setting):
+    """
+    File setting.
+
+    Value of this setting is a pathlib.Path instance.
+    """
+
+    def __init__(self, exists=True, mode="r", *args, **kwargs):
```

Comment:
Custom arguments are added to the end, not beginning.

Reply:
Done.

Do I really have to name all the parameters of the parent class, or is there a nicer way to do it?

Reply:
Nope.